includes/definitions/arubaos.yaml

```diff
@@ -3,8 +3,10 @@ text: ArubaOS
 type: wireless
 icon: aruba
 over:
-    - { graph: device_arubacontroller_numaps, text: 'Number of APs' }
-    - { graph: device_arubacontroller_numclients, text: 'Number of Clients' }
+    - { graph: device_wireless_ap-count, text: 'AP Count' }
+    - { graph: device_wireless_clients, text: 'Client Count' }
+    - { graph: device_arubacontroller_numaps, text: 'Number of APs (Legacy)' }
```

Comment:
Is this worthwhile to leave?  Maybe we should have a check if the rrd exists before displaying the overview graph?

Reply:
@murrant That would be nice indeed !

Reply:
I took a look and it is not feasible with the current code. :(  Would take some major re-writing of graphing.